vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/TreeGridElement.java

```diff
@@ -36,6 +36,30 @@ public void scrollToRowAndWait(int row) {
         waitUntilLoadingFinished();
     }
 
+    /**
+     * Scrolls to the row with the given indexes. The indexes are hierarchical,
+     * starting with the root index.
+     *
+     * @param indexes
+     *            the indexes of the row to scroll to
+     */
+    public void scrollToRowAndWait(int... indexes) {
+        waitUntilLoadingFinished();
+        callFunction("scrollToIndex", indexes);
+        waitUntilLoadingFinished();
+    }
+
+    /**
+     * Scrolls to the row with the given flat index.
+     *
+     * @param row
+     *            the row to scroll to
+     */
+    public void scrollToFlatRowAndWait(int row) {
+        waitUntilLoadingFinished();
+        scrollToFlatRow(row);
```

Comment:
`waitUntilLoadingFinished` at the end is missing?

Reply:
I thought it was unnecessary to add it since scrollToFlatRow already does it after scrolling.